enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/service/AuditService.java

```diff
@@ -32,6 +33,12 @@
 @Service
 public class AuditService {
 
+    private static final String IDENTITY_VERIFICATION_ID = "identityVerificationId";
```

Comment:
I see this declaration for the third time in the review. Perhaps we can share these constants?

Reply:
Sorry, I did the same change in several PRs, but yes, it is a shared constant.